series/praha-pyvo/events/2019-10-16-remotely.yaml

```diff
@@ -0,0 +1,19 @@
+city: praha
+start: 2019-10-16 19:00:00
+name: Pražské Pyvo
+number: 103
+description: |
+  Remotely
```

Comment:
Jakub, I'm sure you can do better than that :D

Reply:
you are really strict! :D